index.html

```diff
@@ -40,6 +40,181 @@ <h1>Hey there. I'm Sainingo <br><span>I'm a software developer</span></h1>
+               <div class="desc">
+                   <h4>Muilti-Post Stories Gain+Glory</h4>
+                   <ul>
+                       <li>Ruby on rails</li>
+                       <li>css</li>
+                       <li>Javascript</li>
+                       <li>html</li>
+                   </ul>
+               </div>
+               <button type="button">See Project</button>
+            </div>
+        </div>
+
+        <!-- fourth container -->
+        <div class="container">
+            <div class="card">
+               <img src="./assets/images/placeholder.png" alt="image placeholder">
+               <div class="desc">
+                   <h4>Muilti-Post Stories Gain+Glory</h4>
+                   <ul>
+                       <li>Ruby on rails</li>
+                       <li>css</li>
+                       <li>Javascript</li>
+                       <li>html</li>
+                   </ul>
+               </div>
+               <button type="button">See Project</button>
+            </div>
+        </div>
+
+        <!-- fifth container -->
+        <div class="container">
+            <div class="card">
+               <img src="./assets/images/placeholder.png" alt="image placeholder" >
+               <div class="desc">
+                   <h4>Muilti-Post Stories Gain+Glory</h4>
+                   <ul>
+                       <li>Ruby on rails</li>
+                       <li>css</li>
+                       <li>Javascript</li>
+                       <li>html</li>
+                   </ul>
+               </div>
+               <button type="button">See Project</button>
+            </div>
+        </div>
+
+        <!-- sixth container -->
+        <div class="container">
+            <div class="card">
+               <img src="./assets/images/placeholder.png" alt="image placeholder" >
+               <div class="desc">
+                   <h4>Muilti-Post Stories Gain+Glory</h4>
+                   <ul>
+                       <li>Ruby on rails</li>
+                       <li>css</li>
+                       <li>Javascript</li>
+                       <li>html</li>
+                   </ul>
+               </div>
+               <button type="button">See Project</button>
+            </div>
+        </div>
+     </section>
+
+     <!-- About me section -->
+     <section>
+         <div class="upper-bg"></div>
+         <div class="about-me">
+            <h4>About Me</h4>
+            <p>I can help you build a product, feature or website look through some of my work and experience!
+                if you like what you see and have a project you need coded, dont hesitate to contact me.
+            </p>
+            <button type="button">Get My Resume</button>
+         </div>
+         <!-- cards -->
+         <div class="card-section">
+             <img src="./assets/images/icon-lang.png" alt="image placeholder">
+             <p>Languages</p>
```

Comment:
- [ ] Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

Reply:
Hi @AnselemOdims. Thanks for taking your time to review my code, Kindly am not understanding that this part? please clarify for me

Reply:
@AnselemOdims still waiting??

Reply:
Hi @AnselemOdims

Reply:
Hi @sainingo, 
This means the languages, frameworks and skills should also be as list items in a list wrapper.
```
<ul>
  <li>Languages</li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>
```

Reply:
👍🏽 Good now.